### DISCUSSION TITLE
Волонтёры

### DISCUSSION THREAD

**Discussion by qwsq4** [ArtemA1ekseev/telegram-bot-animal-shelter#3] [Q&A] [answered]
Будем ли мы делать сервис волонтёров или просто поставим плейсхолдер? Я написал примерную модель волонтера

**Comment by ArtemA1ekseev** [comment, marked as answer]
Привет @qwsq4 Думаю нам нужно для начала, сделать вариант с плейсхолдером. Но если у тебя есть идеи реализации именно готовой модели, я посмтрю на твой вариант.

**Reply by qwsq4** [reply]
Я запушил модель в мою ветку

**Reply by ArtemA1ekseev** [reply]
хорошо.
